src/include/ZividPython/Wrappers.h

```diff
@@ -13,6 +16,50 @@
 
 namespace ZividPython
 {
+    namespace
+    {
+        std::string toSnakeCase(const std::string capitalized)
+        {
+            if(capitalized.empty())
```

Comment:
Wonder if we should also check for spaces (leading, between words,trailing) or any other characters like +.- ....?

Reply:
As long as this function is in this namespace and is not exposed anywhere else, I think we don't need to have tests for all kinds of special characters.